### PR TITLE
[example] Upgrade expo-cli to v4.1.4

### PR DESCRIPTION
**Pull request by bntzio**
Upgrades to `expo-cli` v4.1.4 to fix the example app.

Prior to that version running the example app will throw the following error if users do not belong to the `react-navigation` organization.

Screen Shot 2021-03-01 at 11 58 49 PM

By upgrading to v4.1.4 (see https://github.com/expo/expo-cli/pull/3148) if users do not belong to the org the app fallbacks to offline mode, so they can successfully start the example app.

Screen Shot 2021-03-02 at 12 08 04 AM

Screen Shot 2021-03-02 at 12 12 15 AM

### CODE REVIEW

File: example/tsconfig.json [hunk 1]
```diff
@@ -0,0 +1,16 @@
+{
```
Comment (satya164) [inline example/tsconfig.json:1]: Why is this change necessary?

**Review by satya164 — COMMENTED** [review]
Th

File: example/tsconfig.json (hunk 1 above)
Comment (bntzio) [reply example/tsconfig.json:1]: This it’s automatically generated by expo when you start the app, I believe it finds for a tsconfig and if there’s none, it creates this for you.

**Comment by satya164** [comment]
thanks for the PR. expo-cli is latest version on main branch.
